### DISCUSSION TITLE
[30 Giugno 2023] Pipeline

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/architettura-degli-elaboratori#85] [Pipeline (M-Z)]
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/4ccc3374-55de-4318-9f8f-774e29e75f56)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/0629df80-f5e0-42fb-b8c3-8f6a6fffaf2a)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/16db4350-c62c-4892-a62d-ac8f41d0d526)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/4b12ece9-726c-4d88-87ed-ed4b45a3a682)

**Comment by rimaout** [comment]
![image](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/551e5894-9092-445b-8e9a-e7b19f253523)
![image](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/768715a8-ab7c-431c-a6f7-fb9aadf0a374)
![image](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/241a50a5-c10f-4487-9b3d-91e80c427444)
![image](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/f137dbd2-151a-4662-9461-d49f5b116da8)
